website/docs/tutorials/setup-react.md

```diff
@@ -24,6 +24,10 @@ This setup guide is for any project which uses React.
     npm install --save @lingui/macro @lingui/react
     ```
 
+    :::tip
+    For those who prefer not to use Babel, Lingui offers the [SWC Plugin](/docs/ref/swc-plugin.md) as an alternative.
+    :::
+
```

Comment:
Again, I think it would be more clear to have it directly under the first paragraph of the `Install` section.

Reply:
fixed

Reply:
Im not agree. People don't read docs thoroughly, when they in the beginning will see "install babel macros" they will close page and will go for searching another lib which is not depending on babel. Note about SWC should be in the place where babel is proposed to be installed.

Reply:
yeah, there is a risk to be unnoticed 😬

Reply:
Or, even more obvious we can create 2 parapgraphs in this guide:

1. Setup for Babel projects
    - packages and install steps...
3. Setup for SWC projects 
   - packages and install steps...

Reply:
Or if Docasaurus allow creating tabs, it might be used for organizing this

Reply:
@andrii-bodnar You have moved it to under the last paragraph, not under the first paragraph of the `Install` section.

Reply:
Restructured these installation sections a bit more, please take a look

---

> if Docasaurus allow creating tabs

It is available only in `mdx` syntax, not `md` which is currently used

Reply:
LGTM